ras/sosreport.py

```diff
@@ -38,6 +38,12 @@ def run_cmd_out(cmd):
         return process.system_output(cmd, shell=True, ignore_status=True,
                                      sudo=True).decode("utf-8")
 
+    def run_cmd_search(self, cmd, search_str):
+        self.output = self.run_cmd_out(cmd)
+        for line in reversed(self.output.splitlines()):
+            if search_str in line:
+                return line.strip()
```

Comment:
any specific reason `line[0]` is changed?

Reply:
yes line[0] had empty value. .strip is removing it.